### DISCUSSION TITLE
Version 2.4.0-preview.441 cannot open workflow definition

### DISCUSSION THREAD

**Discussion by tannontawat** [elsa-workflows/elsa-core#2585] [Q&A] [answered]
I use .NET Core 5 and use Elsa version 2.4.0-preview.441. It can build success and show the dashboard, but I click the workflow definition or create new workflow definition its not response.

![image](https://user-images.githubusercontent.com/32186802/145777843-af75f525-4906-4a9d-8657-6ed4c2de24e6.png)

Then I inspect in Google Chrome (F12) I found some script not found with 404 code.

![image](https://user-images.githubusercontent.com/32186802/145777611-295fa1b3-e36e-4aa8-984a-78816947cd2e.png)

How I fix it to working normally???

Best regards.

**Comment by tannontawat** [comment, marked as answer]
Sorry for asking. I find my problem that my browser store old cookies, so I clear all cookies and it work normally.

**Reply by sfmskywalker** [reply]
No worries, thank you for letting us know!
